client/src/components/AddPerk.svelte

```diff
@@ -0,0 +1,98 @@
+<script>
+    import { onMount } from 'svelte';
+    import { goto } from '$app/navigation';
```

Comment:
I don't think app/navigation will be needed anymore, routing can be set up through svelte pages similar to the Home.svelte page navigation.

Reply:
Yeah this PR is blocked by my current PR that loads spring boot locally